src/routes/controllers/project.controller.js

```diff
@@ -58,6 +58,33 @@ const createProject = async (req, res, next) => {
   }
 };
 
+const getProject = async (req, res, next) => {
+  try {
+    const { projectId } = req.params;
+    const { _id: userId } = req.user;
+
+    if (!mongoose.isValidObjectId(projectId)) {
+      const error = createError(400, BAD_REQUEST);
+      logger.error(error.toString());
+
+      return next(error);
+    }
+
+    const project = await Project.findOne({ _id: projectId, owner: userId })
+      .lean()
+      .exec();
+
+    return res.json({
+      ...project,
+      secret_key: getUserSecretKey(project.secret_key),
+    });
```

Comment:
```suggestion
    project.secret_key = getUserSecretKey(project.secret_key)
    return res.json(project);
```
이렇게 응답하면 프론트에서 보여줄 때 편하지 않을까요?

Reply:
좋아요! 이게 더 알아보기도 쉬운 것 같아요 ㅎㅎ 수정하겠습니다!